R/module-helpers.R

```diff
@@ -353,133 +353,3 @@ get_ssp_regions <- function(pcGDP, reg_names, income_group,
 
   regions$region
 }
-
```

Comment:
Ugh. Question: what about creating a new branch from `master`, dropping in all your code (no downstream work, right, so 'just' the main chunk and its xml partner, along with potentially some inputs?), and creating a new PR?

I'm concerned that fixing this is going to be beyond my git-fu.